src/main/webapp/transcript.js

```diff
@@ -0,0 +1,71 @@
+
+const TRANSCRIPT_CONTAINER = 'transcript-container';
+
+/**
+ * Sends a POST request to the transcript.
+ */
+function sendPostToTranscript() {
+  // TODO: Update with user input.
+  const paramsString = 'id=123456789&video=3ymwOvzhwHs';
+  const params = new URLSearchParams(paramsString);
+  fetch('/transcript', {method: 'POST', body: params})
+      .then(fetchTranscriptLines(paramsString));
+}
+
+/**
+ * Fetches the transcript lines from \transcript.
```

Comment:
/transcript

Reply:
Either `@link` to the servlet or `@code` to the constant for the url

Reply:
Thanks for catching that! Added @code for `ENDPOINT_TRANSCRIPT`